packages/marketplace/src/resources/locales/en.json

```diff
@@ -0,0 +1,75 @@
+{
+  "translation": {
+    "settings": {
+      "title": "Settings",
+      "optionsHeading": "Options",
+      "starCountLabel": "Stars count",
+      "tagsLabel": "Tags",
+      "devToolsLabel": "Theme developer tools",
+      "hideInstalledLabel": "Hide installed when browsing",
+      "colourShiftLabel": "Shift colours every minute",
+      "tabsHeading": "Tabs",
+      "resetHeading": "Reset",
+      "resetBtn": "$t(settings.resetHeading)",
```

Comment:
Is this necessary when `resetBtn` essentially uses `resetHeading` string?

Reply:
In case it ever changes. Technically they serve different purposes. One is a label vs an affirmative action. I think it's best practice, but idk.

Reply:
It may differ in different languages so I'm not sure whether to remove it so yeah ig we can keep it so.